src/DotNet/Emit/MethodBodyReaderBase.cs

```diff
@@ -24,6 +24,8 @@ public abstract class MethodBodyReaderBase {
 		protected uint codeEndOffs;
 		/// <summary>Start offset of method</summary>
 		protected uint codeStartOffs;
+		/// <summary>The module context</summary>
+		protected ModuleContext context;
```

Comment:
This should be readonly. I'd also rename the field to `contextOrNull` since I currently don't use NRT in dnlib.

Reply:
The lack of `readonly` is due to the change you commented on above. Several of these fields should ideally be `readonly` in my mind, but I assumed there's some reason why things are the way they are.

Reply:
It's a breaking change to change the other ones to readonly now but this new field can be made readonly.